OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -79,17 +83,7 @@ public MainMenuLogic(Widget widget, World world, ModData modData)
 
 			mainMenu.Get<ButtonWidget>("SINGLEPLAYER_BUTTON").OnClick = () => SwitchMenu(MenuType.Singleplayer);
 
-			mainMenu.Get<ButtonWidget>("MULTIPLAYER_BUTTON").OnClick = () =>
-			{
-				SwitchMenu(MenuType.None);
-				Ui.OpenWindow("MULTIPLAYER_PANEL", new WidgetArgs
-				{
-					{ "onStart", RemoveShellmapUI },
-					{ "onExit", () => SwitchMenu(MenuType.Main) },
-					{ "directConnectHost", null },
-					{ "directConnectPort", 0 },
-				});
-			};
+			mainMenu.Get<ButtonWidget>("MULTIPLAYER_BUTTON").OnClick = () => OpenMultiplayerPanel();
```

Comment:
This can simplify to `mainMenu.Get<ButtonWidget>("MULTIPLAYER_BUTTON").OnClick = OpenMultiplayerPanel;`